### PR TITLE
Update migration_2_5.rst

### PR DESCRIPTION
**Pull request by kix**
Fixed some formatting issues and typos.

### CODE REVIEW

**Comment by Ocramius** [comment]
@kix thanks for the fixes!

File: docs/en/changelog/migration_2_5.rst [hunk 1]
```diff
@@ -320,27 +320,27 @@ Better support for EntityManagerInterface
 Many of the locations where previously only the ``Doctrine\ORM\EntityManager``
 was allowed are now changed to accept the ``EntityManagerInterface`` that was
 introduced in 2.4. This allows you to more easily use the decorator pattern
-to extend the EntityManager if you need. Its still not replaced everywhere,
+to extend the EntityManager if you need. It`s still not replaced everywhere,
```
Comment (stof) [inline docs/en/changelog/migration_2_5.rst]: this should not be a backtick

Comment (Ocramius) [inline docs/en/changelog/migration_2_5.rst]: @stof this was fixed when merging branch `UpgradingDocs`

Comment (kix) [inline docs/en/changelog/migration_2_5.rst]: Oops, sorry. And thanks, @Ocramius!

**Pull request merged**